test/e2e/fixtures.go

```diff
@@ -109,15 +137,7 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error) {
 	return data, nil
 }
 
-func logsDirForTest(testName string) string {
-	// a filepath-friendly timestamp format.
-	const timeFormat = "Jan02-15-04-05"
-	timeStamp := time.Now().Format(timeFormat)
-	logsDir := filepath.Join(testOptions.logsExportDir, fmt.Sprintf("%s.%s", testName, timeStamp))
-	return logsDir
-}
-
-func exportLogs(tb testing.TB, data *TestData) {
+func exportLogs(tb testing.TB, data *TestData, dirName string) {
```

Comment:
for this parameter, I would say that `logsSubDir` would be a better name than `dirName`